auth/api/v1/api.go

```diff
@@ -440,8 +440,7 @@ func (w Wrapper) IntrospectAccessToken(ctx echo.Context) error {
 		Iat:        &iat,
 		Sid:        claims.SubjectID,
 		Service:    &claims.Service,
-		Name:       claims.Name,
```

Comment:
Make sure we don't break demos with this

Reply:
Won't since Name was never used. Dummy method didn't fill anything and IRMA doesn't have this.